scripts/apps/archive/directives/MediaPreview.ts

```diff
@@ -90,6 +91,27 @@ export function MediaPreview(api, $rootScope, desks, superdesk, content, storage
                 return _.map(scope.item.company_codes, 'qcode').join(', ');
             };
 
+            // Check if there is any association for fieldId
```

Comment:
If the comment only repeats what is in the function signature, it doesn't provide any additional information and isn't worth having.